Automattic/wp-calypso
# More elaborate/descriptive error message on the `Use a domain I own` page

#### Changes proposed in this Pull Request

At this moment when a customer attempts to transfer a .nl domain to WordPress.com they get this screen:

![](https://d.pr/i/wHFVHx+)
[Direct Link](https://d.pr/i/wHFVHx+)

`This domain can not be transfered` message is quite generic and should be replaced with a dynamic message that includes their TLD so that the error message is more informative like:
`We don't currently support transfer of .nl domains to WordPress.com.`

![](https://d.pr/i/inWxuO+)
[Direct Link](https://d.pr/i/inWxuO+)

#### Testing instructions

1.  Visit http://wordpress.com/domains/manage/:site
2. Click `Add a domain` button on top right and then click the `Domain I own` drop down.
3. Proceed to search for a .nl based domain and click the `next` button to see two card components, this message would be in the second card component.



Related to #61463
